tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/gmail.py

```diff
@@ -40,7 +40,7 @@ def tap_grant_access(self):
         Wait(self.marionette).until(expected.element_enabled(grant_access))
         grant_access.tap()
         # Go back to displayed Contacts app before waiting for the picker
```

Comment:
You need to add this line here:
from gaiatest.apps.contacts.app import Contacts
because otherwise Contacts isn't recognized.